pkg/ui/workspaces/cluster-ui/src/insights/workloadInsights/util/queriesCell.tsx

```diff
@@ -21,15 +21,15 @@ export function QueriesCell(
   textLimit: number,
 ): React.ReactElement {
   if (
-    !transactionQueries.length ||
-    (transactionQueries.length === 1 &&
-      transactionQueries[0].length < textLimit)
+    !transactionQueries?.length ||
+    (transactionQueries?.length === 1 &&
```

Comment:
Ditto for the addition of `?` here. `!transactionQueries?.length` will ensure that the part after `||` means that transactionQueries has a non-zero length.